src/Components/Search/Search.js

```diff
@@ -11,17 +11,24 @@ const ALBUM_TITLES = gql`
   }
 `
 
-const Search = () => {
+const Search = ({ setSearch }) => {
+
   const [userQuery, setUserQuery] = useState('');
   const { loading, error, data } = useQuery(ALBUM_TITLES);
 
-  const determineOptionStatus = () => {
-    if (loading) return <option value="Loading options..." />
-    if (error) return <option value="Problem loading options!" />
+  // const determineOptionStatus = () => {
+
+  //   if (loading) return <option value="Loading options..." />
+  //   if (error) return <option value="Problem loading options!" />
+
+  //   return data.map(title => {
+  //     return <option value={ title } />
+  //   })
+  // }
 
-    return data.map(title => {
-      return <option value={ title } />
-    })
+  const helpSetSearch = (event) => {
```

Comment:
As discussed, we should update this function identifier to be more descriptive and action-oriented.

Reply:
will brainstorm